From eb3452063cd4c321db3f912dbc9b9e5a0c4a60a7 Mon Sep 17 00:00:00 2001
From: John Stultz <john.stultz@linaro.org>
Date: Wed, 9 Oct 2019 23:12:21 +0000
Subject: [PATCH 571/830] ANDROID: Kconfig.gki: Add extra audio selections
X-NVConfidentiality: public

Add SND_PCM_IEC958 to GKI_HIDDEN_SND_SOC_CONFIGS

Bug: 142268770
Change-Id: I0927c00ef476cb232a8f6c87534cadea62d4a21a
Signed-off-by: John Stultz <john.stultz@linaro.org>
---
 init/Kconfig.gki | 1 +
 1 file changed, 1 insertion(+)

diff --git a/init/Kconfig.gki b/init/Kconfig.gki
index 14265007d7e8..8dc30bfadcdf 100644
--- a/init/Kconfig.gki
+++ b/init/Kconfig.gki
@@ -21,6 +21,7 @@ config GKI_HIDDEN_REGMAP_CONFIGS
 config GKI_HIDDEN_SND_SOC_CONFIGS
 	bool "Hidden SND_SOC configs needed for GKI"
 	select SND_SOC_GENERIC_DMAENGINE_PCM if (SND_SOC && SND)
+	select SND_PCM_IEC958
 	help
 	  Dummy config option used to enable hidden SND_SOC configs.
 	  These are normally selected implicitely when a module
-- 
2.17.1

